pkg/cmd/server/apis/config/v1/conversions.go

```diff
@@ -1,409 +1,262 @@
-		obj.MCSAllocatorRange = "s0:/2"
+
+func Convert_v1_MasterNetworkConfig_To_config_MasterNetworkConfig(in *legacyconfigv1.MasterNetworkConfig, out *internal.MasterNetworkConfig, s conversion.Scope) error {
+	if err := s.DefaultConvert(in, out, conversion.IgnoreMissingFields); err != nil {
+		return err
 	}
-	if obj.MCSLabelsPerProject == 0 {
-		obj.MCSLabelsPerProject = 5
+
+	if len(out.ClusterNetworks) == 0 {
+		out.ClusterNetworks = []internal.ClusterNetworkEntry{
+			{
+				CIDR:             in.DeprecatedClusterNetworkCIDR,
+				HostSubnetLength: in.DeprecatedHostSubnetLength,
+			},
+		}
 	}
-}
-func SetDefaults_IdentityProvider(obj *IdentityProvider) {
-	if len(obj.MappingMethod) == 0 {
-		// By default, only let one identity provider authenticate a particular user
-		// If multiple identity providers collide, the second one in will fail to auth
-		// The admin can set this to "add" if they want to allow new identities to join existing users
-		obj.MappingMethod = "claim"
+
+	if out.VXLANPort == 0 {
+		out.VXLANPort = 4789
 	}
+	return nil
 }
-func SetDefaults_GrantConfig(obj *GrantConfig) {
-	if len(obj.ServiceAccountMethod) == 0 {
-		obj.ServiceAccountMethod = GrantHandlerPrompt
+
+func Convert_v1_AdmissionPluginConfig_To_config_AdmissionPluginConfig(in *legacyconfigv1.AdmissionPluginConfig, out *internal.AdmissionPluginConfig, s conversion.Scope) error {
+	if err := autoConvert_v1_AdmissionPluginConfig_To_config_AdmissionPluginConfig(in, out, s); err != nil {
+		return err
 	}
-}
 
-func addConversionFuncs(scheme *runtime.Scheme) error {
-	return scheme.AddConversionFuncs(
-		convert_runtime_Object_To_runtime_RawExtension,
-		convert_runtime_RawExtension_To_runtime_Object,
-
-		func(in *NodeConfig, out *internal.NodeConfig, s conversion.Scope) error {
-			return s.DefaultConvert(in, out, conversion.IgnoreMissingFields)
-		},
-		func(in *internal.NodeConfig, out *NodeConfig, s conversion.Scope) error {
-			return s.DefaultConvert(in, out, conversion.IgnoreMissingFields)
-		},
-		func(in *KubernetesMasterConfig, out *internal.KubernetesMasterConfig, s conversion.Scope) error {
-			if err := s.DefaultConvert(in, out, conversion.IgnoreMissingFields); err != nil {
-				return err
-			}
+	if len(in.Configuration.Raw) == 0 && (in.Configuration.Object == nil) {
+		out.Configuration = nil
+	} else {
+		if err := convert_runtime_RawExtension_To_runtime_Object(&in.Configuration, &out.Configuration, s); err != nil {
+			return nil
+		}
+	}
 
-			if out.DisabledAPIGroupVersions == nil {
-				out.DisabledAPIGroupVersions = map[string][]string{}
-			}
+	return nil
+}
 
-			// the APILevels (whitelist) needs to be converted into an internal blacklist
-			if len(in.APILevels) == 0 {
-				out.DisabledAPIGroupVersions[internal.APIGroupKube] = []string{"*"}
+func Convert_config_AdmissionPluginConfig_To_v1_AdmissionPluginConfig(in *internal.AdmissionPluginConfig, out *legacyconfigv1.AdmissionPluginConfig, s conversion.Scope) error {
```

Comment:
thank you github for making this impossible to review.

Reply:
Are there any changes in the conversions?

Reply:
> Are there any changes in the conversions?

Yeah. The last four are new. They are used to call a conversion from internal to external on contained runtime.Objects.